google-cloud-bigtable-emulator/src/main/java/com/google/cloud/bigtable/emulator/v2/Emulator.java

```diff
@@ -98,10 +101,24 @@ public synchronized void start() throws IOException, TimeoutException, Interrupt
     }
     this.port = getAvailablePort();
 
+    // Try to align the localhost address across java  & golang emulator
+    // This should fix issues on systems that default  to  ipv4 but the  jvm is started with
```

Comment:
Just a nit: there are a few extra spaces in these 2 lines of comment?

Reply:
fixed